### PR TITLE
Tutorial part 1

### PR DESCRIPTION
**Pull request by munoz0raul**
@mike-scott and @Tim-Anderson , I created this first PR to start getting feedback on how it looks integrated into our docs.

Still need to be done:
 - Estimated Time needs to defined and updated.

### CODE REVIEW

**Comment by doanac** [comment]
Docs for 4608b52b860efedb326b2ebbbf8b8c155c136a97 are browesable at: https://ci.foundries.io/projects/fio-docs/builds/850/docs/artifacts/html/index.html

File: source/tutorials/getting-started-with-docker/index.rst [hunk 1]
```diff
@@ -0,0 +1,584 @@
...
+.. note::
+
+  Estimated Time: 3 minutes
+
+.. _tutorial-gs-with-docker-prerequisite:
+
+Learning Objectives
+-------------------
+
+- Download your container.git repository.
```
Comment (vanmaegima) [inline source/tutorials/getting-started-with-docker/index.rst:20]: container**s**.git

**Review by vanmaegima — APPROVED** [review]
Just a minor comment, other than that it's approved

**Comment by doanac** [comment]
Docs for 726c19a9bdec1a1db5c9d68acf3fd620b5b0b633 are browesable at: https://ci.foundries.io/projects/fio-docs/builds/851/docs/artifacts/html/index.html

**Review by angolini — APPROVED** [review]
I think the order is a little bit confusing. I would prefer the conclusion closer to "Prerequisites and Prework" and all the "commands" more like an appendix.

Another suggestion would be split the tutorial in more than 1 file (only a suggestion)

thanks for the content! Very important

**Comment by doanac** [comment]
Docs for 20ac46ce074f3dca14b5df651af948b98524286b are browesable at: https://ci.foundries.io/projects/fio-docs/builds/852/docs/artifacts/html/index.html

File: source/tutorials/getting-started-with-docker/index.rst [hunk 2]
```diff
@@ -0,0 +1,584 @@
...
+===========================
+
+FoundriesFactory gives you the ability to over-the-air update docker-compose 
+applications, so this page contains step-by-step instructions on how to get 
+started with Docker and docker-compose applications. This tutorial will give you 
+the basic commands and concepts to help you to create your application.
+
+.. note::
+
+  Tutorial Estimated Time: 3 minutes
```
Comment (mike-scott) [inline source/tutorials/getting-started-with-docker/index.rst:13]: Let's change the estimate to something like `10 minutes` and if they get done sooner, then it's a pleasant surprise.

**Review by mike-scott — CHANGES_REQUESTED** [review]

File: source/tutorials/getting-started-with-docker/index.rst (hunk 2 above)
Comment (mike-scott) [reply source/tutorials/getting-started-with-docker/index.rst:13]: Maybe even 15 minutes.

File: source/tutorials/getting-started-with-docker/index.rst [hunk 3]
```diff
@@ -0,0 +1,584 @@
...
+----------
+You have learned the basic commands and structure of Docker and docker-compose 
+application. At this point, if you just created your Factory, the initial build 
+should finish. In the next tutorial, you will start from here, you will commit 
+and push your changes to the remote repository. The CI will then start a new 
+build and will deploy this application on your device.
+
+.. warning::
+
+  If you follow this tutorial before flashing and registering your device,
```
Comment (mike-scott) [inline source/tutorials/getting-started-with-docker/index.rst:578]: If you *followed* this tutorial before flashing and registering your device

**Comment by doanac** [comment]
Docs for 60cf871a75f8ecb221fed760396da32359e9c3c7 are browesable at: https://ci.foundries.io/projects/fio-docs/builds/853/docs/artifacts/html/index.html

**Pull request closed without merging**